### PR TITLE
[docs] Move DreamBooth training materials to docs

### PR DESCRIPTION
**Pull request by stevhliu**
This PR moves relevant DreamBooth training materials from its current GitHub folder to the docs as part of the docs consolidation efforts.

### CODE REVIEW

File: docs/source/en/training/dreambooth.mdx [hunk 1]
```diff
@@ -10,55 +10,66 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
 
-xFormers is not part of the training requirements, but [we recommend you install it if you can](../optimization/xformers). It could make your training faster and less memory intensive.
+xFormers is not part of the training requirements, but we recommend you [install](../optimization/xformers) it if you can because it could make your training faster and less memory intensive.
 
-After all dependencies have been set up you can configure a [🤗 Accelerate](https://github.com/huggingface/accelerate/) environment with:
+After all the dependencies have been set up, initialize a [🤗 Accelerate](https://github.com/huggingface/accelerate/) environment with:
 
 ```bash
 accelerate config
 ```
```
Comment (yiyixuxu) [inline docs/source/en/training/dreambooth.mdx:49]: we don't want to miss this information:

Or for a default accelerate configuration without answering questions about your environment

```bash
accelerate config default
```

Or if your environment doesn't support an interactive shell e.g. a notebook

```python
from accelerate.utils import write_basic_config
write_basic_config()
```

**Review by yiyixuxu — APPROVED** [review]
Awesome! thanks! 
I saw you added a link to the dreambooth blog post - any thoughts on where that article should live in the doc if we want to move it to the doc?

File: docs/source/en/training/dreambooth.mdx (hunk 1 above)
Comment (stevhliu) [reply docs/source/en/training/dreambooth.mdx:49]: Ohh thanks for adding these!

**Comment by stevhliu** [comment]
I think the article could go in the Conceptual Guide section since it helps deepen a user's understanding of how certain hyperparameters affect the DreamBooth results :)

**Pull request merged**